src/hearings/containers/hearing-actuals/hearing-actuals-view-edit-parties/hearing-actuals-view-edit-parties.component.ts

```diff
@@ -270,8 +272,15 @@ export class HearingActualsViewEditPartiesComponent implements OnInit, OnDestroy
             ...this.hearingActualsMainModel.hearingActuals && this.hearingActualsMainModel.hearingActuals.actualHearingDays && this.hearingActualsMainModel.hearingActuals.actualHearingDays[0],
             actualDayParties: actualParties,
           }
-        ],
+        ]
       };
+
+      hearingActuals.actualHearingDays.forEach(item => {
+        if (!item.hearingDate) {
+          item.hearingDate = this.calculateEarliestHearingDate(this.hearingActualsMainModel.hearingPlanned.plannedHearingDays);
+        }
+      });
```

Comment:
avoid assign value in forEach loop like this way, just put that under line 273.
```suggestion
actualHearingDays: [
          {
            ...this.hearingActualsMainModel.hearingActuals && this.hearingActualsMainModel.hearingActuals.actualHearingDays && this.hearingActualsMainModel.hearingActuals.actualHearingDays[0],
            actualDayParties: actualParties,
            hearingDate: this.calculateEarliestHearingDate(this.hearingActualsMainModel.hearingPlanned.plannedHearingDays);
          }
        ]
```